lib/data/schema.js

```diff
@@ -91,6 +91,10 @@ const _recurseFormFields = (instance, bindings, repeats, selectMultiples, envelo
         // if we have no binding node but we have children, assume this is a
         // structural node with no repeat or direct data binding; recurse.
         field.type = 'structure';
+      } else if (tag.name === 'entity' && joinedPath === '/meta/entity') {
```

Comment:
This condition might be overkill, i just wanted to be very clear which tag is targeted here!